static/app/components/forms/controls/textarea.tsx

```diff
@@ -3,13 +3,11 @@ import TextareaAutosize from 'react-autosize-textarea';
 import isPropValid from '@emotion/is-prop-valid';
 import styled from '@emotion/styled';
 
-import {inputStyles} from 'sentry/styles/input';
-import space from 'sentry/styles/space';
+import {InputProps, inputStyles} from 'sentry/components/input';
 
-type InputProps = Omit<Parameters<typeof inputStyles>[0], 'theme'>;
 export interface TextAreaProps
   extends Omit<React.TextareaHTMLAttributes<HTMLTextAreaElement>, 'css'>,
-    InputProps {
+    Omit<InputProps, keyof React.InputHTMLAttributes<HTMLInputElement>> {
```

Comment:
This basically evaluates to any special props added to `Input`, including `monospace`, `nativeSize`, and `size`.